src/test/java/io/cdap/plugin/salesforce/plugin/source/streaming/SalesforceStreamingSourceConfigTest.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.salesforce.plugin.source.streaming;
+
+import com.sforce.soap.partner.PartnerConnection;
+import com.sforce.soap.partner.QueryResult;
+import com.sforce.soap.partner.sobject.SObject;
+import com.sforce.ws.ConnectionException;
+import io.cdap.plugin.salesforce.SObjectDescriptor;
+import io.cdap.plugin.salesforce.SObjectFilterDescriptor;
+import io.cdap.plugin.salesforce.SalesforceQueryUtil;
+import io.cdap.plugin.salesforce.plugin.OAuthInfo;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import java.util.ArrayList;
+
+/**
+ * Tests for SalesforceStreamingSourceConfig
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({SalesforceStreamingSourceConfig.class, SalesforceStreamingSource.class, SObjectDescriptor.class,
+        SalesforceQueryUtil.class, SObjectFilterDescriptor.class})
+public class SalesforceStreamingSourceConfigTest {
+
```

Comment:
please fix indent in all files. See https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard

Reply:
Indentation has been fixed as per the CDAP Coding standard.